py-polars/tests/unit/test_rows.py

```diff
@@ -14,19 +14,15 @@ def test_row_tuple() -> None:
 
     # return row by index as namedtuple
```

Comment:
Comment referring to `namedtuple`. Should that be chaged?

Reply:
Good one, fixed!